examples/species/1-propanol_UMVT/input.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# APE variables
+
+coordinate_system = "Normal Mode" # coordinate_system include "Normal Mode", "E-Optimized" and "E'-Optimized"
+cut_off_energy = 0.05 # Use 0.05 hartrees as cut_off_energy to ensure each mode sample to 4 natural lengths in each direction
+step_size_factor = 1 # step_size = step_size_factor * natural_length
+number_of_natural_length = 4 # sample till number_of_natural_length * natural_length in each direction of each mode
+coordinate_type = "DLC" # coordinate type include "DLC", "HDLC" and "TRIC" 
```

Comment:
Another stuff we should correct is the option of `DLC` should be renamed as `RIC`.